session-1/coba.html

```diff
@@ -0,0 +1,159 @@
+                <div class="col-md-6 col-sm-6 py-3 align-self-center">
+                    <h2 class="display-5 fs-1 fw-bold">What's my experience?</h2>
+                    <p class="fs-3">Describe your experience in 20 words or more. Such as Insternship experience, Bootcamp, Student Exchange and so on.</p>
+                </div>
+                <div class="col-md-6 col-sm-6 py-3">
+                    <div class="row">
+                        <div class="col-12 col-md-6 shadow p-3 mb-5 bg-body rounded bg-color-light-gray mt-5 pt-3 w-100 box-card box our-services">
+                            <div class="card-title">
+                                <h3 class="fw-bold">UI/UX Designer</h3>
+                                <figure>
+                                    <blockquote class="blockquote">
+                                        <p>PT.Persentologic Indonesia · Internship</p>
+                                    </blockquote>
+                                    <p class="blockquote-footer fs-5">January 2022 - Present</p>
+                                    <p class="fs-5 lh-sm">User Experience researcher and create user interfaces for PT.Persentologic Indonesia landing pages.
+                                    </p>
+                                </figure>
+                            </div>
+                        </div>
+                        <div class="col-12 col-md-6 shadow p-3 mb-5 bg-body rounded bg-color-light-gray mt-5 pt-3 w-100 box-card box our-services">
+                            <div class="card-title">
+                                <h3 class="fw-bold">Graphic Designer</h3>
+                                <figure>
+                                    <blockquote class="blockquote">
+                                        <p>Upwork · Freelance</p>
+                                    </blockquote>
+                                    <p class="blockquote-footer fs-5">January 2021 - Present</p>
+                                    <ul class="fs-5 lh-sm">
+                                        <li>Making Logo for Company, Group such as E- Sports, Bussiness, and Personal user.</li>
+                                        <li>Making Avatar or Vector Illustration.</li>
+                                        <li>Designing Pamphlet, Banner, Info Graphics, etc.</li>
+                                    </ul>
+                                </figure>
+                            </div>
+                        </div>
+                        <div class="col-12 col-md-6 shadow p-3 mb-5 bg-body rounded bg-color-light-gray mt-5 pt-3 w-100 box-card box our-services">
+                            <div class="card-title">
+                                <h3 class="fw-bold">Front-End Web Developer</h3>
+                                <figure>
+                                    <blockquote class="blockquote">
+                                        <p>Nomura Research Institute · Contract</p>
```

Comment:
Separate Company name and Job Type (e.g Internship, contract) into different tag. You can use span.